src/d3a/models/myco_matcher/external_matcher.py

```diff
@@ -35,24 +35,32 @@ def publish_offers_bids(self, message):
         published data are of the following format
         market_offers_bids_list_mapping = {"market_id" : {"bids": [], "offers": [] }, }
         """
-        # TODO: message can contain filters
-        data = {"event": "offers_bids_response"}
+        response_data = {"event": "offers_bids_response"}
+        data = json.loads(message.get("data"))
+        filters = data.get("filters", {})
+        # IDs of markets the client is interested in
+        market_ids_in_interest = filters.get("markets", None)
```

Comment:
Nit: I find this name a bit difficult to interpret. Do you think that any of these would be a potential alternative?
- `relevant_market_ids`
- `considered_market_ids`
- `filtered_market_ids`
- `requested_market_ids`

Reply:
```suggestion
        market_ids_in_interest = filters.get("markets", [])
```
If you change the default to an empty list, you could simplify the conditional below:
```python
if area_uuid not in market_ids_in_interest:
```

Reply:
Actually this can break the logic 
I do need to check the if filtered_market_ids, in case the client did not pass it or it's empty, I don't want to do the filter (In this case nothing will be returned whereas this is supposed to return all the records)